it/mail/chrome/messenger/addressbook/addressBook.properties

```diff
@@ -40,44 +40,44 @@
 #
 # The following are used by the Mailing list dialog
 #
-emptyListName=Fornire un nome per la lista.
+emptyListName=Inserisci il nome della lista.
 lastFirstFormat=%S, %S
-firstLastFormat=%S %S  
+firstLastFormat=%S %S
 
-newCardTitle=New Card  
+newCardTitle=Nuovo contatto
 # %S will be the card's display name
-newCardTitleWithDisplayName=New Card for %S
-editCardTitle=Edit Card
+newCardTitleWithDisplayName=Nuovo contatto per %S
+editCardTitle=Modifica contatto
 # %S will be the card's display name
-editCardTitleWithDisplayName=Edit Card for %S
+editCardTitleWithDisplayName=Modifica contatto per %S
 # don't translate vCard
 editVCardTitle=Modifica vCard
 # %S will be the card's display name, don't translate vCard
 editVCardTitleWithDisplayName=Modifica vCard per %S
 
 ## LOCALIZATION NOTE (cardRequiredDataMissingMessage):  do not localize \n
-cardRequiredDataMissingMessage=È indispensabile inserire almeno uno fra questi campi:\nE-mail, Nome, Cognome, Nome visualizzato, Organizzazione.
-cardRequiredDataMissingTitle=Mancano delle informazioni indispensabili
-incorrectEmailAddressFormatMessage=L'indirizzo di e-mail principale deve essere nella forma utente@dominio.
+cardRequiredDataMissingMessage=Devi inserire almeno uno fra questi campi:\nE-mail, Nome, Cognome, Nome visualizzato, Organizzazione.
+cardRequiredDataMissingTitle=Non sono presenti informazioni obbligatorie
```

Comment:
CORRECTIONS NEEDED: I would keep the first part from the original version and the second part from the new version, as follows: 

 +cardRequiredDataMissingTitle=Mancano informazioni obbligatorie